tracer/src/Datadog.Trace.SourceGenerators/TagsListGenerator/Sources.cs

```diff
@@ -80,27 +82,61 @@ partial class ")
             {
                 foreach (var property in tagList.MetricProperties)
                 {
+                    var tagBytes = Encoding.UTF8.GetBytes(property.TagValue);
+                    var tagArray = new string[tagBytes.Length];
+                    for (var i = 0; i < tagBytes.Length; i++)
+                    {
+                        tagArray[i] = tagBytes[i].ToString();
+                    }
+
+                    var tagByteArray = string.Join(", ", tagArray);
+
                     sb.Append(
                            @"
-        private static readonly byte[] ")
+        // ")
                       .Append(property.PropertyName)
                       .Append(@"Bytes = Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8.GetBytes(""")
```

Comment:
This isn't technically correct anymore.
```suggestion
                      .Append(@"Bytes = System.Text.Encoding.UTF8.GetBytes(""")
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2753/commits/9badd3251f935b4c305c24b948d4f3a5393b4798